pkg/types/azure/doc.go

```diff
@@ -2,5 +2,8 @@
 // configuration and management.
 package azure
 
-// Name is name for the Azure platform.
+// Name is the name for the Azure platform.
 const Name string = "azure"
+
+// StackName is the name for the Azure Stack platform.
+const StackName string = "azurestack"
```

Comment:
I am concerned that this constant here is going to add to confusion making it seem like azurestack is a separate platform.
```suggestion
// StackTerraformName is the name used for the terraform code when installing to the Azure Stack cloud environment.
const StackTerraformName string = "azurestack"
```